ci/infra/testrunner/tests/test_dockercaps.py

```diff
@@ -45,6 +45,16 @@
     containers:
     - name: app
       image: registry.suse.de/suse/containers/sle-server/15/containers/suse/sle15:15.1
+      command: ['/bin/sh', '-c', 'sleep 3600']
+---
+apiVersion: v1
+kind: Pod
+metadata:
+    name: sle15sp2
+spec:
+    containers:
+    - name: app
+      image: registry.suse.de/suse/containers/sle-server/15/containers/suse/sle15:15.2
       command: ['/bin/sh', '-c', 'sleep 3600']"""
```

Comment:
nit
```suggestion
      command: ['/bin/sh', '-c', 'sleep 3600']
      """
```

Reply:
It is changing how it was done before. I personally don't really care either way, but I need to update commit message, or create a separate PR for it, or convince that the end-of-file style with newline is better in some regard. 

I have very little energy for this.